interop.py

```diff
@@ -130,12 +130,12 @@ def _check_impl_is_compliant(self, name: str) -> bool:
             "DOWNLOADS=" + downloads_dir.name + " "
             'SCENARIO="simple-p2p --delay=15ms --bandwidth=10Mbps --queue=25" '
             "CLIENT=" + self._implementations[name]["image"] + " "
-            "docker-compose up --timeout 0 --abort-on-container-exit -V sim client"
+            "docker-compose up --timeout 0 --exit-code-from client -V sim client"
```

Comment:
We’re starting multiple containers, and we can’t know which one exits first.

Reply:
Why does sim container exit before client container?  I think a compliant client exits first with exit code 127, then docker-compose kills sim.